code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -161,6 +161,14 @@
 	max_ammo = 8
 	multiple_sprites = 2
 
+/obj/item/ammo_box/magazine/pdwdap
+	name = "PDW magazine (5.8x21mm DAP)"

```

Comment:
This should be called 5.8x22mm, or 5.8x21mm, whichever this gun is meant to use.